### ISSUE TITLE
Android release crash RN-0.60.5 = E/unknown:NativeViewHierarchyManager: Unable to update properties for view tag 115

### ISSUE THREAD

**Issue by Frindecent** [facebook/react-native#26398]
React Native version:
System:
    OS: macOS 10.14.5
    CPU: (4) x64 Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz
    Memory: 131.55 MB / 8.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 11.11.0 - /usr/local/bin/node
    Yarn: 1.13.0 - /usr/local/bin/yarn
    npm: 6.10.3 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 12.4, macOS 10.14, tvOS 12.4, watchOS 5.3
  IDEs:
    Android Studio: 3.4 AI-183.6156.11.34.5692245
    Xcode: 10.3/10G8 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.8.6 => 16.8.6 
    react-native: 0.60.5 => 0.60.5 
  npmGlobalPackages:
    react-native-cli: 2.0.1
    react-native-create-bridge: 2.0.1

## Steps To Reproduce

I have built a native View in android side and i am importing it in react-native. The native view works well in debug version as desired. After i test it by
 react-native run-android --variant=release 
The native module produces following error:
```
2019-09-10 22:59:38.659 27970-27970/? E/unknown:NativeViewHierarchyManager: Unable to update properties for view tag 115
    com.facebook.react.uimanager.IllegalViewOperationException: ViewManager for tag 115 could not be found.
     View already dropped? false.
    Last index 0 in last 100 views[I@8d7bf36
        at com.facebook.react.uimanager.NativeViewHierarchyManager.resolveViewManager(NativeViewHierarchyManager.java:112)
        at com.facebook.react.uimanager.NativeViewHierarchyManager.updateProperties(NativeViewHierarchyManager.java:135)
        at com.facebook.react.uimanager.UIImplementation.synchronouslyUpdateViewOnUIThread(UIImplementation.java:309)
        at com.facebook.react.uimanager.UIManagerModule.synchronouslyUpdateViewOnUIThread(UIManagerModule.java:396)
        at com.facebook.react.animated.PropsAnimatedNode.restoreDefaultValues(PropsAnimatedNode.java:71)
        at com.facebook.react.animated.NativeAnimatedNodesManager.restoreDefaultValues(NativeAnimatedNodesManager.java:336)
        at com.facebook.react.animated.NativeAnimatedModule$18.execute(NativeAnimatedModule.java:350)
        at com.facebook.react.animated.NativeAnimatedModule$2.execute(NativeAnimatedModule.java:141)
        at com.facebook.react.uimanager.UIViewOperationQueue$UIBlockOperation.execute(UIViewOperationQueue.java:530)
        at com.facebook.react.uimanager.UIViewOperationQueue$1.run(UIViewOperationQueue.java:844)
        at com.facebook.react.uimanager.UIViewOperationQueue.flushPendingBatches(UIViewOperationQueue.java:952)
        at com.facebook.react.uimanager.UIViewOperationQueue.access$2200(UIViewOperationQueue.java:44)
        at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.doFrameGuarded(UIViewOperationQueue.java:1012)
        at com.facebook.react.uimanager.GuardedFrameCallback.doFrame(GuardedFrameCallback.java:29)
        at com.facebook.react.modules.core.ReactChoreographer$ReactChoreographerDispatcher.doFrame(ReactChoreographer.java:172)
        at com.facebook.react.modules.core.ChoreographerCompat$FrameCallback$1.doFrame(ChoreographerCompat.java:84)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:947)
        at android.view.Choreographer.doCallbacks(Choreographer.java:761)
        at android.view.Choreographer.doFrame(Choreographer.java:693)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:935)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7050)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
2019-09-10 22:59:38.660 27970-27970/? E/unknown:NativeViewHierarchyManager: Unable to update properties for view tag 2967
    com.facebook.react.uimanager.IllegalViewOperationException: ViewManager for tag 2967 could not be found.
     View already dropped? false.
    Last index 0 in last 100 views[I@8d7bf36
        at com.facebook.react.uimanager.NativeViewHierarchyManager.resolveViewManager(NativeViewHierarchyManager.java:112)
        at com.facebook.react.uimanager.NativeViewHierarchyManager.updateProperties(NativeViewHierarchyManager.java:135)
        at com.facebook.react.uimanager.UIImplementation.synchronouslyUpdateViewOnUIThread(UIImplementation.java:309)
        at com.facebook.react.uimanager.UIManagerModule.synchronouslyUpdateViewOnUIThread(UIManagerModule.java:396)
        at com.facebook.react.animated.PropsAnimatedNode.restoreDefaultValues(PropsAnimatedNode.java:71)
        at com.facebook.react.animated.NativeAnimatedNodesManager.restoreDefaultValues(NativeAnimatedNodesManager.java:336)
        at com.facebook.react.animated.NativeAnimatedModule$18.execute(NativeAnimatedModule.java:350)
        at com.facebook.react.animated.NativeAnimatedModule$2.execute(NativeAnimatedModule.java:141)
        at com.facebook.react.uimanager.UIViewOperationQueue$UIBlockOperation.execute(UIViewOperationQueue.java:530)
        at com.facebook.react.uimanager.UIViewOperationQueue$1.run(UIViewOperationQueue.java:844)
        at com.facebook.react.uimanager.UIViewOperationQueue.flushPendingBatches(UIViewOperationQueue.java:952)
        at com.facebook.react.uimanager.UIViewOperationQueue.access$2200(UIViewOperationQueue.java:44)
        at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.doFrameGuarded(UIViewOperationQueue.java:1012)
        at com.facebook.react.uimanager.GuardedFrameCallback.doFrame(GuardedFrameCallback.java:29)
        at com.facebook.react.modules.core.ReactChoreographer$ReactChoreographerDispatcher.doFrame(ReactChoreographer.java:172)
        at com.facebook.react.modules.core.ChoreographerCompat$FrameCallback$1.doFrame(ChoreographerCompat.java:84)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:947)
        at android.view.Choreographer.doCallbacks(Choreographer.java:761)
        at android.view.Choreographer.doFrame(Choreographer.java:693)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:935)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7050)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
2019-09-10 22:59:38.660 27970-27970/? E/unknown:NativeViewHierarchyManager: Unable to update properties for view tag 2947
    com.facebook.react.uimanager.IllegalViewOperationException: ViewManager for tag 2947 could not be found.
     View already dropped? false.
    Last index 0 in last 100 views[I@8d7bf36
        at com.facebook.react.uimanager.NativeViewHierarchyManager.resolveViewManager(NativeViewHierarchyManager.java:112)
        at com.facebook.react.uimanager.NativeViewHierarchyManager.updateProperties(NativeViewHierarchyManager.java:135)
        at com.facebook.react.uimanager.UIImplementation.synchronouslyUpdateViewOnUIThread(UIImplementation.java:309)
        at com.facebook.react.uimanager.UIManagerModule.synchronouslyUpdateViewOnUIThread(UIManagerModule.java:396)
        at com.facebook.react.animated.PropsAnimatedNode.restoreDefaultValues(PropsAnimatedNode.java:71)
        at com.facebook.react.animated.NativeAnimatedNodesManager.restoreDefaultValues(NativeAnimatedNodesManager.java:336)
        at com.facebook.react.animated.NativeAnimatedModule$18.execute(NativeAnimatedModule.java:350)
        at com.facebook.react.animated.NativeAnimatedModule$2.execute(NativeAnimatedModule.java:141)
        at com.facebook.react.uimanager.UIViewOperationQueue$UIBlockOperation.execute(UIViewOperationQueue.java:530)
        at com.facebook.react.uimanager.UIViewOperationQueue$1.run(UIViewOperationQueue.java:844)
        at com.facebook.react.uimanager.UIViewOperationQueue.flushPendingBatches(UIViewOperationQueue.java:952)
        at com.facebook.react.uimanager.UIViewOperationQueue.access$2200(UIViewOperationQueue.java:44)
        at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.doFrameGuarded(UIViewOperationQueue.java:1012)
        at com.facebook.react.uimanager.GuardedFrameCallback.doFrame(GuardedFrameCallback.java:29)
        at com.facebook.react.modules.core.ReactChoreographer$ReactChoreographerDispatcher.doFrame(ReactChoreographer.java:172)
        at com.facebook.react.modules.core.ChoreographerCompat$FrameCallback$1.doFrame(ChoreographerCompat.java:84)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:947)
        at android.view.Choreographer.doCallbacks(Choreographer.java:761)
        at android.view.Choreographer.doFrame(Choreographer.java:693)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:935)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7050)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
```

the nativeView was working allright until 0.60. but now has failed to load only in released version and not in debug version.

Snack, code example, screenshot, or link to a repository:

My JS side code : 
```
import {requireNativeComponent, ViewPropTypes, PropTypes} from 'react-native';

const propTypes = {
    someRandomProp: "PropTypes.string",
    someRandomType: "PropTypes.string",
};


const CustomView = requireNativeComponent('CustomViewReactManager');

export default CustomView;
```
and then using this CustomView in another file as

```
import CustomView from './CustomView.js';

export default class AnsweredAndroid extends React.Component {
render(){
<CustomView style={{flex:1}} />
}
}
```

1. the above code works as a charm in debug version but app crashes in release version.
2. It was working allright in release version before 0.60.5

**Comment by Frindecent** [comment]
looks like there was an issue inside bundling the app and the view wasnt being bundled. closing!

**Comment by MarkRihter** [comment]
> looks like there was an issue inside bundling the app and the view wasnt being bundled. closing!

I faced the same issue, can you describe the solution?

**Comment by cvvspb** [comment]
@Frindecent I'm observing the same issue. Could you elaborate a little bit?

**Comment by bakerac4** [comment]
Im also seeing this same issue. Everything is correctly rendered, then I go to another screen and come back and I see this error and a blank screen.

**Comment by leelandclay** [comment]
I found this while trying to solve the same error. Since I couldn't find anything specific, I started removing anything that would change the state and force a re-render. I eventually got to a point where I noticed a warning about key not being set on items in a list. I found a spot within my render where I was iterating over a state object and forgot to add the key. I added it and the error went away.

My thoughts are that the render statement tried to line up the items and freaked out...but that's just a guess
